File: back/src/main/java/com/d101/back/dto/response/OcrResponse.java
package com.d101.back.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;


@NoArgsConstructor
@AllArgsConstructor
@Data
public class OcrResponse {
    private int calorie;
    private double carbohydrate;
    private double protein;
    private double fat;
}

File: front/D101/app/src/main/java/com/ssafy/d101/model/OCRResponse.kt
package com.ssafy.d101.model

data class OCRResponse (
    val calorie : Int,
    val carbohydrate : Double,
    val sugar : Double,
    val protein : Double,
    val fat : Double,
    val saturated : Double,
    val trans : Double,
    val cholesterol : Double,
    val sodium : Double
)